frontend/src/components/Package.jsx

```diff
@@ -33,7 +33,7 @@ export default function Package({ data, showRequest }) {
         <div className="flex flex-row flex-wrap justify-center items-center">
           <span className="flex flex-col inline-block bg-gray-200 rounded-[5rem] px-10 py-10 text-md font-semibold text-gray-700 mr-2 min-w-[50%] w-[fit-content]">
             {data.selectedItems.map((item) => (
-              <span>
+              <span key={item.itemCount + item.itemName}>
```

Comment:
nit: this works fine, but itemNames (should) be unique already; i.e. items cannot be entered into selectedItems twice. at the moment though, nothing stops the user from inserting an existing item into the inventory
```suggestion
              <span key={item.itemName}>
```